packages/run-protocol/src/vpool-xyk-amm/multipoolMarketMaker.js

```diff
@@ -249,6 +242,9 @@ const start = async (zcf, privateArgs) => {
     );
     return zcf.getIssuerForBrand(brand);
   };
+  const poolStorageNode = await E(privateArgs.storageNode)?.getChildNode(
+    `pool${secondaryBrandToPool.getSize()}`,
```

Comment:
@Chris-Hibbert any concerns with this? I think downstream needs this PR quickly so if you recommend changes here and this lands without them I'll make sure to follow up.

Reply:
> the way of generating a unique name for the child node

👍 

> The AMM never deletes pools, so this number grows monotonically.

Perfect.

> Are the poolStorageNodes themselves durable? Will we need to be able to recreate or re-find the names or pools on revival?

I believe storage nodes are durable and I think we landed on them being part of durable object state. Though they're also cheap to re-create if needed.